source/developers/references/developer_guide/process/FAQ-about-pull-requests.rst

```diff
@@ -0,0 +1,178 @@
+request, the Community PM responsible for the PR, and the last date the
+Community PM checked in on it. To do this, find the "Projects" section, and
+click on the "⌄" caret in the top right corner to see a display like this:
+
+.. image:: /_images/devguide_pr_status.png
+
+============================
+FAQs about your Pull Request
+============================
+
+**My pull request has been waiting for review for a few weeks, is something
+wrong?**
+
+No! We receive hundreds of pull requests from the community (which is awesome!),
+but that means sometimes the process to get PRs through the review process and
+merged might take a bit of time due to things like:
+
+* The reviewing team is at capacity and needs to review the pull request during a later sprint
+* Related work or dependencies may be blocking a PR's ability to be merged
+
+The Community PMs will do our best to make sure all updates related to delays
+are communicated within the PR. Additionally, if the reviewing team needs
+additional information or context for your contribution, they will ask within
+the PR.
+
+**There haven't been any updates in my PR in a few weeks. How do you handle
+stalled PRs?**
+
+The Community Project Managers proactively stay on top of pull requests, and
+will ping reviewers for updates, actions, etc. when things seem stalled. If you
+have any questions, you can tag the assigned Community PM for help / updates in
+the PR at any time.
+
+**I submitted a PR a while ago, something came up, and I am not able to work on
+it for the foreseeable future. What happens now?**
+
+If a PR is stalled on the author's side, the Community PMs will ping the author
+a few times before labeling the PR as “inactive” after a month or so.
+
+* If you want to pursue the PR, but don't have capacity at the moment, please
+  let us know directly in the PR if possible. At that point we can leave it as a
+  draft, or we can close it if needed, and can reopen it at a later time if
+  you're able to pursue it.
+* If an author is unresponsive for a month or more, or hasn't had capacity to
+  come back to check on the PR, the Community PMs may need to close the PR due
+  to inactivity (though, again, it can be reopened at a later time if needed).
+
+**My PR was labeled as “blocked” - what does that mean?**
+
+Occasionally, a pull request may get blocked by other on-going or related work.
+The Community PMs will stay on top of it to make sure it gets unblocked as soon
+as possible, but note that it may take a bit depending on what's blocking it.
+Updates from the Community PMs and reviewing team should be made in the PR so
+you have updated information.
+
+**My PR was closed and not merged - why?**
+
+There may be times where a pull request is closed (instead of merged) by an
+owning team because the update may no longer be needed, it's duplicative of
+other work, or other reasons. The reason for closing the pull request should be
```

Comment:
@sarina That reminds me -- I think the "duplicate" label is missing from the section on Contribution Board Labels.

Reply:
Can you provide a description for "duplicate"?